eth/validation.py

```diff
@@ -198,7 +198,7 @@ def validate_unique(values, title="Value"):
         )
 
 
-def validate_block_number(block_number, title="Block Number"):
+def validate_block_number(block_number):
```

Comment:
Instead of removing the `title` from here, can you change the inner calls to pass the `title` down?